### PR TITLE
Symfony 4.0 support

### PR DESCRIPTION
**Pull request by umpirsky**
Part of [CV-60699](https://chesscom.atlassian.net/browse/CV-60699).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,9 +16,9 @@
     "require": {
         "php": ">=5.5",
         "jms/serializer": "~1.5",
-        "symfony/routing": "~2.8|~3.0",
-        "symfony/yaml": "~2.8|~3.0",
-        "symfony/validator": "~2.8|~3.0",
+        "symfony/routing": "~2.8|~3.0|~4.0",
```
Comment (ikonic89) [inline composer.json]: Do we really need to have 2-4 here? Can't we remove the `2.8`?

Comment (umpirsky) [reply composer.json]: We can.

Comment (ikonic89) [reply composer.json]: I'll leave it up to you :) Looks good

Comment (umpirsky) [reply composer.json]: Done.

**Pull request merged**